plsql/ExcelGen.pkb

```diff
@@ -1757,7 +1760,7 @@ create or replace package body ExcelGen is
             else
               data.varchar2_value := to_char(data.number_value, 'TM9', NLS_PARAM_STRING);
             end if;
-            stream_write(stream, '<c r="'||cellRef||'"><v>'||data.varchar2_value||'</v></c>');
+            stream_write(stream, '<c r="'||cellRef||'" s="'||to_char(numXfId)||'"><v>'||data.varchar2_value||'</v></c>');
```

Comment:
Minor optimization here, the "s" attribute could be added only when numXfId != 0.

Reply:
Where DEFAULT_NUM_FMT is used (1681,1922) I added comments you may not want.